index.html

```diff
@@ -31,6 +31,84 @@ <h1>ToDoリスト</h1>
 
     <script type="text/javascript">
 
+      const toDoListArea = document.getElementById('todolistarea');
+      const task = document.getElementById('task');
+      const taskList = [];
+
+        document.getElementById('btn').addEventListener('click', () => {
+
+          //タスクをオブジェクトとして指定、クリックごとに配列に追加
+          const newTask = {taskName:task.value,taskStatus:'作業中'};
```

Comment:
`,`と`:`の後ろにはスペースを空けましょう。
`{}`の中の前後にもスペースを開ける現場が多いです。
```suggestion
          const newTask = { taskName: task.value, taskStatus: '作業中' };
```